gatorgrouper_django/gatorgrouper/models.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+
+# Create your models here.
+class Professor(models.Model):
+    email = models.EmailField(max_length = 200)
+    professor_id = models.AutoField(primary_key=True)
+    first_name = models.CharField(max_length = 30)
+    last_name = models.CharField(max_length = 30)
+
+    def __str__(self):
+        return '{}'.format(self.email)
+
+
+class Semester_Class(models.Model):
+    professor_id = models.ForeignKey(
+            Professor,
+            on_delete=models.CASCADE,
+            )
+    semester = models.CharField(max_length = 50)
+    class_id = models.AutoField(primary_key=True)
+    department = models.CharField(max_length = 10)
+    class_number = models.CharField(max_length = 10)
+    class_section = models.CharField(max_length = 10)
```

Comment:
As discussed in our meeting, I would suggest adding an attribute for a domain name so that (eventually) anyone with the link can join the class when signed in with a matching email address

Reply:
Added a domain_name attribute! With a max length of 100 👍